extension/src/popup/components/manageAssets/SearchAsset/styles.scss

```diff
@@ -23,16 +23,12 @@
 
   &__loader {
     position: absolute;
-    top: calc(50% - var(--SearchAsset--loader--dimension));
```

Comment:
this fixes an issue across adding trustlines and soroban tokens where the spinning loader wasn't aligned properly